### PR TITLE
Fixing #61

### PR DESCRIPTION
**Pull request by AndreMaz**
Fixing #61

### CODE REVIEW

File: packages/moleculer-db-adapter-mongo/package.json [hunk 1]
```diff
@@ -29,14 +29,14 @@
   "devDependencies": {
     "benchmarkify": "2.1.0",
     "coveralls": "3.0.2",
-    "eslint": "5.1.0",
+    "eslint": "^5.7.0",
```
Comment (icebob) [inline packages/moleculer-db-adapter-mongo/package.json]: Please use exact version numbers in all package.json. Thanks

**Comment by icebob** [comment]
By the way, it means, the ID is stored as `_id` in DB, but make the conversion before & after save?

**Comment by AndreMaz** [comment]
Yeah, that's the idea. By design [MongoDB](https://docs.mongodb.com/manual/reference/glossary/#term-id) and [NeDB](https://github.com/louischatriot/nedb#inserting-documents) use `_id` to store a document in DB.

**Comment by icebob** [comment]
But this solution is correct with mongoose & sequelize, as well?

**Comment by AndreMaz** [comment]
Now that you are saying... No, it may not work with sequelize as you can freely define the idField during the creation of the Model. There is no `_id` restriction as happens with MongoDB. 

So, you should reject this PR

**Comment by AndreMaz** [comment]
Does it make sense to have an idField option for sequelize?

**Comment by icebob** [comment]
I think we have two option:
1. add a note to sequelize adapter readme that `idField` is not working with it.
2. move idField handling to the adapters instead of moleculer-db, because it seems the ID handling is different in the adapters.

What do you think?

**Comment by AndreMaz** [comment]
Hey @icebob 
I've moved the `idField` handlers into the adapters.

Check it out. If you merge it I'll update the docs with a note regarding the sequelize

**Comment by icebob** [comment]
Sorry, but yesterday I've worked on this project and it caused conflict in package.json & package-lock files. I've updated all dependencies, please overwrite your these files from the master. Thanks.

**Comment by AndreMaz** [comment]
Np :smile:

**Comment by AndreMaz** [comment]
@icebob can you please publish new version on npm?

`npm i https://github.com/moleculerjs/moleculer-db` does not work

**Comment by icebob** [comment]
Yes, tonight I will do it.

**Comment by icebob** [comment]
Released all packages.

**Pull request merged**